packages/flatpak.rb

```diff
@@ -0,0 +1,92 @@
+require 'package'
+
+class Flatpak < Package
+  description 'Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux.'
+  homepage 'https://flatpak.org'
+  @_ver = '1.10.1'
+  version @_ver
+  compatibility 'all'
+  source_url "https://github.com/flatpak/flatpak/releases/download/#{@_ver}/flatpak-#{@_ver}.tar.xz"
+  source_sha256 'c70215792b7cbece83c489dab86adc9bfaf9b140c506affe2a48c92afa3d69b7'
```

Comment:
Should one scroll back in flatpak's commits they'll notice that the source_sha256 field has switched back and forth between c70215792b7cbece83c489dab86adc9bfaf9b140c506affe2a48c92afa3d69b7 and
c1354f42bf3b5d51aeb4028c9b62fd4ffc673ef2ff6e583c17777f5dafdbdcb7. I'm not sure what's going on here but it appears github is giving two different files with very minor differences when requesting for the tarball? I'm not entirely sure but hopefully we can accommodate it.

Reply:
Not sure why that's happening either.

Reply:
This hasn't happened in a bit...